testresults/src/org/labkey/testresults/view/rundown.jsp

```diff
@@ -255,7 +256,11 @@
                         <% for (String test : problemTests) { %>
                         <tr>
                             <td style="width: 200px; max-width: 200px; overflow: hidden; text-overflow: ellipsis; padding: 0;">
+                                <% if (Arrays.stream(problemRuns).anyMatch(run -> problems.isFail(run, test))) { %>
                                 <a href="<%=h(new ActionURL(TestResultsController.ShowFailures.class, c).addParameter("end", df.format(selectedDate)).addParameter("failedTest", test))%>" target="_blank"><%=h(test)%></a>
```

Comment:
You could replace this with
`<%=link(test).href(new ActionURL(TestResultsController.ShowFailures.class, c).addParameter("end", df.format(selectedDate)).addParameter("failedTest", test)).target("_blank")%>`
Add `.clearClasses()` if you don't like the default link style.

Reply:
Thanks, I changed it.